app/services/google_direction_service.rb

```diff
@@ -1,19 +1,14 @@
 class GoogleDirectionService
 
+  extend Connectable
+
   def self.get_travel_time(origin, destination)
-    response = conn.get('directions/json?') do |req|
+    con = conn('https://maps.googleapis.com/maps/api', name: 'key', key: ENV['GOOGLE_API_KEY'] )
```

Comment:
using `extend` instead of `include` because 'Connectable' contains a class method (`conn`)